doc/src/guides/solving/solve-numerically.md

```diff
@@ -0,0 +1,216 @@
+
+To solve for complex roots of real functions, specify a nonreal (either purely
+imaginary, or complex) initial point:
+
+```py
+>>> from sympy import nsolve
+>>> from sympy.abc import x
+>>> nsolve(x**2 + 2, 1) # Real initial point returns no root
+Traceback (most recent call last):
+    ...
+ValueError: Could not find root within given tolerance. (4.18466446988997098217 > 2.16840434497100886801e-19)
+Try another starting point or tweak arguments.
+>>> from sympy import I
+>>> nsolve(x**2 + 2, I) # Imaginary initial point returns a complex root
+1.4142135623731*I
+>>> nsolve(x**2 + 2, 1 + I) # Complex initial point returns a complex root
+1.4142135623731*I
+```
+
+### Ensure the Root Found is in a Given Interval
+
+It is not guaranteed that nsolve will find the root closest to the initial
+point. Here, even though the root `-1` is closer to the initial point of `-0.1`,
+nsolve finds the root `1`:
+
+```py
+>>> from sympy import nsolve
+>>> from sympy.abc import x
+>>> nsolve(x**2 - 1, -0.1)
+1.00000000000000
+```
+
+You can ensure the root found is in a given interval, if such a root exists,
+using `solver='bisect'` by specifying the interval in a tuple. Here, specifying
+the interval `(-10, 0)` ensures that the root `-1` is found:
+
+```py
+>>> from sympy import nsolve
+>>> from sympy.abc import x
+>>> nsolve(x**2 - 1, (-10, 0), solver='bisect')
+-1.00000000000000
+```
+
+## Solve a System of Equations Numerically
+
+### Solve Multidimensional Functions
+
+To solve multidimensional functions, supply a tuple of
+- functions `(f1, f2)`
+- variables to solve for `(x1, x2)`
+- starting values `(-1, 1)`
+
+```py
+>>> from sympy import Symbol, nsolve
+>>> x1 = Symbol('x1')
+>>> x2 = Symbol('x2')
+>>> f1 = 3 * x1**2 - 2 * x2**2 - 1
+>>> f2 = x1**2 - 2 * x1 + x2**2 + 2 * x2 - 8
+>>> print(nsolve((f1, f2), (x1, x2), (-1, 1)))
+Matrix([[-1.19287309935246], [1.27844411169911]])
+```
+
+### Increase Precision of the Solution
+
+You can increase the precision of the solution using `prec`:
+
+```py
+>>> from sympy import Symbol, nsolve
+>>> x1 = Symbol('x1')
+>>> x2 = Symbol('x2')
+>>> f1 = 3 * x1**2 - 2 * x2**2 - 1
+>>> f2 = x1**2 - 2 * x1 + x2**2 + 2 * x2 - 8
+>>> print(nsolve((f1, f2), (x1, x2), (-1, 1), prec=25))
+Matrix([[-1.192873099352460791205211], [1.278444111699106966687122]])
+```
+
+### Create a Function That Can Be Solved With SciPy
+
+As noted above, SymPy focuses on symbolic computation and is not optimized for
+numerical calculations. A recommend use case for SymPy in a workflow is as given
+below:
+- use SymPy to generate (by symbolically simplifying or solving an equation) the
+  mathematical expression
+- use a numerical library such as SciPy to generate numerical solutions
+
+```py
+>>> from sympy import simplify, cos, sin
+>>> from sympy.abc import x, y
+>>> expr = cos(x * (x + x**2)/(x*sin(y)**2 + x*cos(y)**2 + x))
+>>> simplify(expr)
+cos(x*(x + 1)/2)
+>>> from sympy.utilities.lambdify import implemented_function
```

Comment:
I don't see the point of using `implemented_function` here. I guess this is coming from its use in the `lambdify` docstring, but I don't really think it's necessary there either. `implemented_function` is not something that I think most people really need to use.